### PR TITLE
Take the last added loadbalancer, not the first

### PR DESCRIPTION
**Pull request by joedborg**
Addressing https://github.com/juju-solutions/bundle-canonical-kubernetes/issues/712

### CODE REVIEW

**Comment by joedborg** [comment]
[kubeconfig should contain the VIP when keepalived used with kubeapi-load-balancer #712](https://trello.com/c/PZntcF4l/1098-kubeconfig-should-contain-the-vip-when-keepalived-used-with-kubeapi-load-balancer-712)

File: cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py [hunk 1]
```diff
@@ -936,10 +936,12 @@ def addons_ready():
 def loadbalancer_kubeconfig(loadbalancer, ca, client):
```
Comment (ktsakalozos) [inline cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py]: Why is this method called both by the reactive framework and by our code directly?

File: cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py [hunk 2]
```diff
@@ -936,10 +936,12 @@ def addons_ready():
...
     # Get the potential list of loadbalancers from the relation object.
     hosts = loadbalancer.get_addresses_ports()
+    # Use the most recently enrolled loadbalancer.
```
Comment (ktsakalozos) [inline cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py]: Is there a way to get "None" from any of the following getters?

**Comment by Cynerva** [comment]
I'm not a fan of this as a solution. I don't think there's any guarantee that keepalived will be the last host.

This strikes me as more of a documentation issue. If the operator removes the relation between `kubernetes-master:loadbalancer` and `kubeapi-load-balancer:loadbalancer`, then the master should pick the correct address, right?

**Comment by joedborg** [comment]
@Cynerva thanks, I'll double check that's the case (removing the relation)

**Comment by joedborg** [comment]
Yeah, `juju remove-relation kubernetes-master:loadbalancer kubeapi-load-balancer:loadbalancer` seems to fix this

**Pull request closed without merging**